analyzers/its/sources/Net5/Net5/S3440.cs

```diff
@@ -11,5 +11,25 @@ public void Foo()
                 6 => 6
             };
         }
+
+        private int? f;
+        public int Property
```

Comment:
_Nitpick:_ Missing EOL
```suggestion
        private int? f;
        
        public int Property
```